libs/checkpoint-postgres/src/index.ts

```diff
@@ -250,7 +252,7 @@ export class PostgresSaver extends BaseCheckpointSaver {
       pending_sends: [],
     };
     if ("channel_values" in serialized) {
-      delete serialized.channel_values;
```

Comment:
I believe that we actually do want `delete` here - otherwise the key is still in the object with a value of `undefined` and it gets serialized as `null`.